query.py

```
@@ -0,0 +1,69 @@
+import lancedb
+
+# from sentence_transformers import SentenceTransformer
+
+# 1. 加载向量模型（本地 HuggingFace）
```

Ruff (0.14.14)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)